module/VuFind/src/VuFind/Db/Row/User.php

```diff
@@ -369,8 +369,8 @@ public function getSavedData(
      * Add/update a resource in the user's account.
      *
      * @param \VuFind\Db\Row\Resource $resource        The resource to add/update
-     * @param \VuFind\Db\Row\UserList $list            The list to store the resource
-     * in.
+     * @param int                     $list            The list to store the resource
+     *                                                 in.
```

Comment:
Normally when param comments exceed a single line, we don't need to indent the next line. However, in this particular case, the best solution would be to unwrap it to a single line, since our new 120-character line length limit allows this to fit on a single line.